script.js

```diff
@@ -1,4 +1,114 @@
+//Project 2: Beat That (Base+Comfortable)
+
+var rollDice = "ROLL DICE";
+var chooseSequence = "CHOOSE SEQUENCE";
+var gameMode = rollDice;
+var player1;
+var player2;
+var currentPlayer = 1;
```

Comment:
Nice! Having this as a global variable helps us to reduce repetition in the main function